src/main/scala/GameLib.scala

```diff
@@ -1,33 +1,72 @@
 package strategygames
 
 sealed abstract class GameLib {
-  //def white: Color
-  //def black: Color
   def id: Int
   def name: String
 
   override def toString = s"Lib($name)"
 }
 
 object GameLib {
+
   final case class Chess() extends GameLib {
-    //def white = Color.Chess(chess.White)
-    //def black = Color.Chess(chess.Black)
     def id = 0
     def name = "Chess"
   }
+
   final case class Draughts() extends GameLib {
-    //def white = Color.Draughts(draughts.White)
-    //def black = Color.Draughts(draughts.Black)
     def id = 1
     def name = "Draughts"
   }
 
-  def all = List(Draughts, Chess)
+  def all = List(Chess, Draughts)
 
   // TODO: I'm sure there is a better scala way of doing this
   def apply(id: Int): GameLib = id match {
-    case 0 => Chess()
-    case _ => Draughts()
+    case 1 => Draughts()
+    case _ => Chess()
+  }
+}
+
+sealed abstract class DisplayLib {
+  def id: Int
+  def name: String
+  def shortName: String
+  def codeLib: GameLib
```

Comment:
Just trying to separate out DisplayLib and GameLib more clearly. Perhaps GameLib should be called CodeLib as now we have two. But also happy to change it to `gameLib` and see your point. James can have casting vote...

Reply:
I agree, I think they should be called the same name, gameLib is probably more meaningful than codeLib. I.e. the library for the game.

Reply:
Yeah, I don't mind which one we use, I just want it to be consistent.  RuleFamily, GameFamily etc are all possible. I went for GameLib originally, because it was intended to distinguish between the two 'libs', because other classifications were murky

Reply:
Ok I'll get it changed